docs/maps/valleyNB.md

```diff
@@ -0,0 +1,38 @@
+# Valley (NB)
+
+***
+
+#### ![valleyNBicon](../assets/icons/valleyNB-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.7.0
+- **Description:** A flat plain with mountainous range surrounding it. Has A prebuilt base from the map [Valley](Valley).
```

Comment:
"Has A prebuilt base from the map [Valley](Valley)." sounds like Valley has a prebuilt base. Please reword this

Reply:
A mountainous range surrounding a flat plans with a prebuilt base.